src/components/dev-dot-content/mdx-components/mdx-hcp-callout/mdx-callout.module.css

```diff
@@ -0,0 +1,66 @@
+.root {
+	width: 100%;
+	background: var(--black);
+	color: var(--white);
+	border-radius: 5px;
+	display: flex;
+	position: relative;
+	overflow: hidden;
+	box-shadow: var(--token-elevation-mid-box-shadow);
+	margin: 32px 0;
+
+	&:hover {
+		box-shadow: var(--token-elevation-high-box-shadow);
+
+		& .ctaIcon {
+			transform: translateX(2px);
+		}
+	}
+
+	& .textContainer {
+		padding: 16px 22px;
+	}
+
+	& .heading {
+		/* !important is necessary here to stop leaks from enclosing
+     @hashicorp/react-content component styles */
+		margin: 0 !important;
+	}
+
+	& .subHeading {
+		/* !important is necessary here to stop leaks from enclosing
+     @hashicorp/react-content component styles */
+		margin: 6px 0 16px !important;
+	}
```

Comment:
Honest Q, do we know if these are still necessary? I believe most of our content styles are now scoped to custom components: https://github.com/hashicorp/dev-portal/blob/main/src/components/dev-dot-content/dev-dot-content.module.css

Reply:
Ahh you're right we don't need `important` any more